_posts/2016-02-10-solita-developer-manifesto.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: Solita Developer Manifesto
+author: Kauko
+excerpt: An attempt at putting to words what makes Solita a great place to work, and what is expected of developers, new and old alike.
+tags:
+- Culture
+- Solita
+---
+
+The Agile Manifesto, first published in 2001, is a document that is familiar to many. Aiming to “uncover better ways to develop software”, it defines a set of principles written in the form of “X over Y”, meaning that “while there is value in the items on the right, we value the items on the left more.” This way of presenting the thoughts of the manifesto is not only concise yet expressive, but also helps drive home the point that the manifesto is about prioritization, not about elimination. 
+
+At Solita we place a very large emphasis on the importance of workplace culture. Great culture stems from recruiting the right kinds of people, and then encouraging those people, and giving them the tools, to keep making Solita one of the best places to work in Finland. Perhaps in accordance with the Agile Manifesto, Solita does not have a set of dusty corporate documents defining the required code of conduct, but rather we refer to our four **Core Values** *(passion, boldness, casualness, caring)* that were identified by the employees themselves. Maybe due to the growth that Solita has been blessed to experience in the last few years, a handful of more senior developers wanted to create a document to complement the Core Values. This document is called the *Solita Developer Manifesto*, and, as the name would suggest, its audience are us, the developers.
+
+Reviewing the Principles
+---
+
+The Solita Developer Manifesto is comprised of four principles. With the help of these principles, the manifesto wants to help developers, juniors and seniors alike, understand and agree what is expected of them. Originally they were written in Finnish, so naturally some of the nuances may have been lost in translation.
+
+#### Learning new things and techniques over sticking to what you know
+#### Professional growth over staying in your comfort zone
+Probably the thing I love about Solita the most, is the fact that we’re constantly encouraged to share information and help our colleagues learn. One of my criteria when I was looking for a job some time ago, was that I wanted to join a company where I could feel like I can learn from my coworkers, and this is a promise that Solita has fulfilled hands down. For example, each of our three offices organizes monthly intra-office meetings, where the topic is somehow related to functional programming (dubbed *FP Monthlies*). These are just a drop in the bucket, as sometimes it feels like I would literally have no time to work on my project if I was to accept every invite to every meeting.
```

Comment:
"feels like I would literally have no time to work on my project if I was to accept every invite to every meeting." 
- ei mitenkään väärin, mutta miltä tämä versio kuulostaa?  "feels like I literally wouldn't have time for my project if I participated in every learning session."